include/criterion/internal/assert.h

```diff
@@ -128,16 +144,21 @@ CR_END_C_API
             Fail();                                                         \
     } while (0)
 
-# define cr_fail(Fail, ...)                                     \
-    CR_EXPAND(cr_assert_impl(                                   \
-            Fail,                                               \
-            0,                                                  \
-            dummy,                                              \
-            CRITERION_ASSERT_MSG_FAIL,                          \
-            (),                                                 \
-            __VA_ARGS__                                         \
+
+# define cr_fail(Fail, ...)                                                 \
+    CR_EXPAND(cr_assert_impl(                                               \
+            Fail,                                                           \
+            0,                                                              \
+            dummy,                                                          \
+            CRITERION_ASSERT_MSG_FAIL,                                      \
+            (),                                                             \
+            __VA_ARGS__                                                     \
     ))
 
+# define cr_skip(...)                                                       \
+    CR_SKIP_ABORT_("" __VA_ARGS__);
```

Comment:
No need to define `CR_SKIP_ABORT_` anymore, just call the function directly.
Also, there's a trailing semicolon here.